### PR TITLE
[PUBDEV-8921] Update performance-and-prediction.rst [nocheck]

### PR DESCRIPTION
**Pull request by xiaochi-liu**
typo

### CODE REVIEW

File: h2o-docs/src/product/performance-and-prediction.rst [hunk 1]
```diff
@@ -1620,15 +1620,15 @@ Examples:
 
         # set the predictors and response columns:
         predictors <- c("economy", "cylinders", "displacement", "power", "weight")
-        response = "acceleration"
+        response = "acceleration" # Stopping metric cannot be mean_per_class_error for this continous target variable
```
Comment (mn-mikke) [inline h2o-docs/src/product/performance-and-prediction.rst:1623]: Good catch! @hannah-tillman can turn the example to a classification problem? (E.g.: https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/training_frame.html)

**Review by mn-mikke — COMMENTED** [review]
@xiaochi-liu Thanks a lot for your PR, can you rebase it against `rel-zygmund` so we could get the change out asap?

**Comment by xiaochi-liu** [comment]
Hello @mn-mikke, could you please let me know how to do the rebase?

**Comment by mn-mikke** [comment]
Hi @xiaochi-liu,
the below sequence of commands should get your change aligned with `rel-zygmund` branch:
```
git checkout rel-zygmund
git branch -D patch-4
git checkout -b patch-4
git cherry-pick 2539c098e7fbca48c4d746680cb444b0308fc9f4
git cherry-pick 02aaad3d5b642010457550adf38481d6052254d7
git push -f
```

**Review by valenad1 — COMMENTED** [review]
I have addopted PR [here](https://github.com/h2oai/h2o-3/pull/6461), implement changes from @mn-mikke, and fix example

**Comment by valenad1** [comment]
Will be merged in https://github.com/h2oai/h2o-3/pull/6461

**Pull request closed without merging**